diff --git a/dist/build/swc/options.js b/dist/build/swc/options.js
index 59ec8f63b70e69b196d731d5e9253dcb1e358437..70368be4ed16425a4b69ec2e78be9ec47863ce6d 100644
--- a/dist/build/swc/options.js
+++ b/dist/build/swc/options.js
@@ -48,7 +48,7 @@ function getParserOptions({ filename, jsConfig, ...rest }) {
         ...rest,
         syntax: hasTsSyntax ? "typescript" : "ecmascript",
         dynamicImport: true,
-        decorators: enableDecorators,
+        decorators: true,
         // Exclude regular TypeScript files from React transformation to prevent e.g. generic parameters and angle-bracket type assertion from being interpreted as JSX tags.
         [hasTsSyntax ? "tsx" : "jsx"]: !isTSFile,
         importAssertions: true
@@ -92,6 +92,7 @@ function getBaseSWCOptions({ filename, jest, development, hasReactRefresh, globa
                 } : {},
                 legacyDecorator: enableDecorators,
                 decoratorMetadata: emitDecoratorMetadata,
+                decoratorVersion: "2022-03",
                 useDefineForClassFields: useDefineForClassFields,
                 react: {
                     importSource: (jsConfig == null ? void 0 : (_jsConfig_compilerOptions4 = jsConfig.compilerOptions) == null ? void 0 : _jsConfig_compilerOptions4.jsxImportSource) ?? ((compilerOptions == null ? void 0 : compilerOptions.emotion) && !isReactServerLayer ? "@emotion/react" : "react"),
