diff --git a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/ListQueryBuilder.ts b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/ListQueryBuilder.ts
index 84e48c2..f583dbc 100644
--- a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/ListQueryBuilder.ts
+++ b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/ListQueryBuilder.ts
@@ -1,3 +1,4 @@
+// @ts-ignore
 import { StaticUtils } from "simple-common-utils";
 
 type Key = string;
diff --git a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/Uris.ts b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/Uris.ts
index f9fd8d1..2c793e0 100644
--- a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/Uris.ts
+++ b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/Uris.ts
@@ -1,3 +1,4 @@
+// @ts-ignore
 import { ArrayStringifier } from "simple-common-utils";
 import { stringifyQueryParameters } from "./utils";
 
diff --git a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/uploaders/MultipartUploader.ts b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/uploaders/MultipartUploader.ts
index 0c4b3c0..bfb007c 100644
--- a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/uploaders/MultipartUploader.ts
+++ b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/uploaders/MultipartUploader.ts
@@ -1,7 +1,9 @@
 import {
   ArrayStringifier,
   StaticUtils
+  // @ts-expect-error
 } from 'simple-common-utils'
+// @ts-expect-error
 import utf8 from 'utf8'
 import Uploader from './Uploader'
 import Fetcher, { BodyType, FetchResultType } from '../Fetcher'
diff --git a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/utils.ts b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/utils.ts
index b62deb1..c1942e9 100644
--- a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/utils.ts
+++ b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/aux/utils.ts
@@ -2,6 +2,7 @@ import { toByteArray } from "base64-js";
 import {
   ArrayStringifier,
   StaticUtils
+  // @ts-expect-error
 } from "simple-common-utils";
 
 export function blobToByteArray(blob: Blob): Promise <Uint8Array | null> {
diff --git a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/permissions/Permissions.ts b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/permissions/Permissions.ts
index 8abc330..c61ecda 100644
--- a/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/permissions/Permissions.ts
+++ b/node_modules/@robinbobin/react-native-google-drive-api-wrapper/api/permissions/Permissions.ts
@@ -8,12 +8,14 @@ export default class Permissions extends GDriveApi {
     return new Fetcher(this)
       .setBody(JSON.stringify(requestBody), MimeTypes.JSON)
       .setMethod("POST")
+      // @ts-expect-error
       .fetch(Uris.permissions({fileId, queryParameters}), "json");
   }
 
   delete(fileId: string, permissionId: string, queryParameters?: object) {
     return new Fetcher(this)
       .setMethod("DELETE")
+      // @ts-expect-error
       .fetch(Uris.permissions({fileId, permissionId, queryParameters}), "text");
   }
 };
