diff --git a/node_modules/userop/dist/client.d.ts b/node_modules/userop/dist/client.d.ts
index 1485116..ba79924 100644
--- a/node_modules/userop/dist/client.d.ts
+++ b/node_modules/userop/dist/client.d.ts
@@ -14,4 +14,8 @@ export declare class Client implements IClient {
         userOpHash: string;
         wait: () => Promise<import("./typechain/EntryPoint").UserOperationEventEvent | null>;
     }>;
+    execUserOperation(op: import("./types").IUserOperation, opts?: ISendUserOperationOpts): Promise<{
+        userOpHash: string;
+        wait: () => Promise<import("./typechain/EntryPoint").UserOperationEventEvent | null>;
+    }>;
 }
diff --git a/node_modules/userop/dist/client.js b/node_modules/userop/dist/client.js
index a3a1ffa..9f0721d 100644
--- a/node_modules/userop/dist/client.js
+++ b/node_modules/userop/dist/client.js
@@ -38,6 +38,39 @@ class Client {
             return builder.buildOp(this.entryPoint.address, this.chainId);
         });
     }
+    
+    execUserOperation(op, opts) {
+        var _a;
+        return __awaiter(this, void 0, void 0, function* () {
+            const dryRun = Boolean(opts === null || opts === void 0 ? void 0 : opts.dryRun);
+            (_a = opts === null || opts === void 0 ? void 0 : opts.onBuild) === null || _a === void 0 ? void 0 : _a.call(opts, op);
+            const userOpHash = dryRun
+                ? new context_1.UserOperationMiddlewareCtx(op, this.entryPoint.address, this.chainId).getUserOpHash()
+                : (yield this.provider.send("eth_sendUserOperation", [
+                    (0, utils_1.OpToJSON)(op),
+                    this.entryPoint.address,
+                ]));
+            return {
+                userOpHash,
+                wait: () => __awaiter(this, void 0, void 0, function* () {
+                    if (dryRun) {
+                        return null;
+                    }
+                    const end = Date.now() + this.waitTimeoutMs;
+                    const block = yield this.provider.getBlock("latest");
+                    while (Date.now() < end) {
+                        const events = yield this.entryPoint.queryFilter(this.entryPoint.filters.UserOperationEvent(userOpHash), Math.max(0, block.number - 100));
+                        if (events.length > 0) {
+                            return events[0];
+                        }
+                        yield new Promise((resolve) => setTimeout(resolve, this.waitIntervalMs));
+                    }
+                    return null;
+                }),
+            };
+        });
+    }
+    
     sendUserOperation(builder, opts) {
         var _a;
         return __awaiter(this, void 0, void 0, function* () {
