diff --git a/node_modules/secretjs/src/query/compute.ts b/node_modules/secretjs/src/query/compute.ts
index 4db6491..ccc546a 100644
--- a/node_modules/secretjs/src/query/compute.ts
+++ b/node_modules/secretjs/src/query/compute.ts
@@ -196,7 +196,8 @@ export class ComputeQuerier {
       }
 
       return JSON.parse(fromUtf8(fromBase64(fromUtf8(decryptedBase64Result))));
-    } catch (err) {
+    } catch (e) {
+      const err = e as Error;
       try {
         const errorMessageRgx =
           /encrypted: (.+?): (?:instantiate|execute|query|reply to) contract failed/g;
@@ -216,7 +217,8 @@ export class ComputeQuerier {
           //@ts-ignore
           // return the error string
           return fromUtf8(fromBase64(fromUtf8(decryptedBase64Error)));
-        } catch (parseError) {
+        } catch (parseE) {
+          const parseError = parseE as Error;
           if (parseError.message === "Invalid base64 string format") {
             //@ts-ignore
             // return the error string
diff --git a/node_modules/secretjs/src/secret_network_client.ts b/node_modules/secretjs/src/secret_network_client.ts
index f4e8de0..19d6a1d 100644
--- a/node_modules/secretjs/src/secret_network_client.ts
+++ b/node_modules/secretjs/src/secret_network_client.ts
@@ -1022,7 +1022,8 @@ export class SecretNetworkClient {
       return tx_response
         ? this.decodeTxResponse(tx_response, ibcTxOptions)
         : null;
-    } catch (error) {
+    } catch (e) {
+      const error = e as Error
       if (error?.message === `tx not found: ${hash}`) {
         return null;
       } else {